lib/lookerUtils.ts

```diff
@@ -30,11 +30,14 @@ export function getExperimentLookerDashboardDate(
 export function getLookerSubmissionTimestampDateFilter(
```

Comment:
If you could add `@param` blocks for each parameter and explain how they affect the expected outcomes, this would allow future code readers to able to tell if a specific code behavior is a bug and more easily understand how to call it.